### DISCUSSION TITLE
Application approved but no student benefit package

### DISCUSSION THREAD

**Discussion by snipernan** [community/community#67588] [GitHub Education] [answered]
### Hi everyone,

I have receiced the message 'Congrats! Your request for [@snipernan](https://github.com/snipernan) was approved and your benefits will be available soon' for about 2 days but i still don't have the student benefit package. How long do you usually have to wait?

**Comment by priyanujgogoi-28** [comment, marked as answer]
Chill up mate! Student benefits are usually given out to users between 24-72 hours after they are approved (in most cases). Sometimes you might be delayed for a day or two but eventually you will receive it.

PS: In my case, I received my student benefits on my 3rd day after getting approved.
